saleor/graphql/payment/tests/test_payment.py

```diff
@@ -435,6 +445,78 @@ def test_create_payment_for_checkout_with_active_payments(
     assert active_payments.first().pk not in previous_active_payments_ids
 
 
+@pytest.mark.parametrize("store", [None, "NONE", "ON_SESSION", "OFF_SESSION"])
```

Comment:
Values should be taken from the enum.
```suggestion
@pytest.mark.parametrize("store", [None, StoreEnum.NONE, StoreEnum.ON_SESSION, StoreEnum.OFF_SESSION])
```